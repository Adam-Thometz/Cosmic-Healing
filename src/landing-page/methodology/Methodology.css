.Methodology {
  margin: 5% 0;
}

.Methodology-methods {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-evenly;
  align-items: top;
  width: 90%;
  margin: 0 auto;
}