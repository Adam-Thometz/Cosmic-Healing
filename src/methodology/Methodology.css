.Methodology-methods {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-evenly;
  align-items: top;
  width: 75%;
  margin: 0 auto;
}