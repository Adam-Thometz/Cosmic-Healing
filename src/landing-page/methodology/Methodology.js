import React from 'react';

import './Methodology.css';

import Method from './method-card/Method';
import SectionHeader from '../../_components/section-header/SectionHeader';
import SmallCaption from '../../_components/small-caption/SmallCaption';

import methods from './methods';

const Methodology = () => {
  return (
    <section className='Methodology'>
      <SmallCaption text="HOW I CAN HELP.." />
      <SectionHeader text="My Process" />
      <div className='Methodology-methods'>
        {methods.map(m => <Method
          name={m.name}
          img={m.img}
          description={m.description}
        />)}
      </div>
    </section>
  );
};

export default Methodology;