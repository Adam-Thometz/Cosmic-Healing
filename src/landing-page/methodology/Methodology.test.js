import React from "react";
import { render } from "@testing-library/react";
import Methodology from './Methodology';

describe('Methodology component', () => {
  it('renders without crashing', () => {
    render(<Methodology />);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<Methodology />);
    expect(asFragment()).toMatchSnapshot();
  });
});