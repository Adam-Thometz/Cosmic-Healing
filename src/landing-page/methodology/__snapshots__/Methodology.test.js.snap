// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Methodology component matches the snapshot 1`] = `
<DocumentFragment>
  <section
    class="Methodology"
  >
    <small
      class="SmallCaption"
      style="color: black;"
    >
      HOW I CAN HELP..
    </small>
    <h2
      class="SectionHeader"
    >
      My Process
    </h2>
    <div
      class="Methodology-methods"
    >
      <figure
        class="Method"
      >
        <img
          alt="Dreamwork"
          src="dreamwork.png"
        />
        <figcaption>
          <h3>
            Dreamwork
          </h3>
          <p>
            Decode symbols within your dreams to uncover subconscious messages
          </p>
        </figcaption>
      </figure>
      <figure
        class="Method"
      >
        <img
          alt="Shadow Work"
          src="shadow-work.png"
        />
        <figcaption>
          <h3>
            Shadow Work
          </h3>
          <p>
            Integrate the repressed and hidden parts of you
          </p>
        </figcaption>
      </figure>
      <figure
        class="Method"
      >
        <img
          alt="Emotional Processing"
          src="emotional-processing.png"
        />
        <figcaption>
          <h3>
            Emotional Processing
          </h3>
          <p>
            Understand and process your emotions from the past and present
          </p>
        </figcaption>
      </figure>
      <figure
        class="Method"
      >
        <img
          alt="Motivational Interviewing"
          src="motivational-interviewing.png"
        />
        <figcaption>
          <h3>
            Motivational Interviewing
          </h3>
          <p>
            Find internal motivation to change self-destructive behavior
          </p>
        </figcaption>
      </figure>
      <figure
        class="Method"
      >
        <img
          alt="Cognitive Behavioral"
          src="cognitive-behavioral.png"
        />
        <figcaption>
          <h3>
            Cognitive Behavioral
          </h3>
          <p>
            Release negative patterns and limiting beliefs
          </p>
        </figcaption>
      </figure>
      <figure
        class="Method"
      >
        <img
          alt="Person Centered"
          src="person-centered.png"
        />
        <figcaption>
          <h3>
            Person Centered
          </h3>
          <p>
            Gain the confidence to take charge of your own life
          </p>
        </figcaption>
      </figure>
    </div>
  </section>
</DocumentFragment>
`;
